next.config.mjs

```diff
@@ -95,8 +92,11 @@ const config = {
         ".json",
       ],
     },
+    staticGenerationRetryCount: 2,
+    staticGenerationMaxConcurrency: 20,
+    staticGenerationMinPagesPerWorker: 30,
   },
-  productionBrowserSourceMaps: true,
+  expireTime: 3600,
```

Comment:
what does this prop do? I couldn't find a mention of it in the docs

Reply:
this is cache stale time - https://nextjs.org/docs/app/api-reference/config/next-config-js/expireTime

Reply:
![image](https://github.com/user-attachments/assets/0cc7d609-e564-4028-8d9e-3a6a77bb9def)

Reply:
can you add a comment for it + link, it looks like it is used to set the cache headers for CDNs as opposed to modify the ISR revalidate time

Reply:
done